### PR TITLE
Added topk() support from v3.6.0.

### PR DESCRIPTION
**Pull request by mlloreda**

### CODE REVIEW

File: arrayfire/library.py [hunk 1]
```diff
@@ -422,6 +422,14 @@ class CANNY_THRESHOLD(_Enum):
     MANUAL = _Enum_Type(0)
     AUTO_OTSU = _Enum_Type(1)
 
+class TOPK(_Enum):
+    """
+    Top-K function types
```
Comment (9prady9) [inline arrayfire/library.py]: topk function `filter options` ? may be, types is confusing I think.

File: arrayfire/statistics.py [hunk 2]
```diff
@@ -108,3 +108,38 @@ def corrcoef(x, y):
...
+    ----------
+
+    data: af.Array
+          Input array to return k elements from.
+
+    k: scalar. default: 0
+       The number of elements to return from input array.
+
+    dim: optional: scalar. default: 0
+         The dimension along which the top k elements are extracted.
```
Comment (9prady9) [inline arrayfire/statistics.py]: Can you add the note about dim=0 support here as well. It is there in tests but not in the documentation.

File: arrayfire/library.py (hunk 1 above)
Comment (pavanky) [reply arrayfire/library.py]: This is similar to what the enum is in C/C++. Let us keep this consistent.

Comment (pavanky) [reply arrayfire/library.py]: Sorry I misread the location of this comment. I agree with @9prady9 here about fixing the docs.

Comment (mlloreda) [reply arrayfire/library.py]: @9prady9 @pavanky Done.

File: arrayfire/statistics.py (hunk 2 above)
Comment (mlloreda) [reply arrayfire/statistics.py]: Done.

**Comment by 9prady9** [comment]
@mlloreda Can you please resolve the conflict after rebasing from master(I merged another PR which is where the conflicts is coming from)

**Comment by mlloreda** [comment]
@9prady9 Done.

**Pull request merged**